package.json

```
@@ -1,59 +1,41 @@
 {
-  "name": "@readme/eslint-config",
-  "version": "8.5.1",
-  "description": "ReadMe coding standards",
-  "main": "index.js",
-  "author": "Jon Ursenbach <jon@ursenba.ch>",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:readmeio/eslint-config.git"
-  },
+  "name": "standards",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Core coding and style standards for all ReadMe technology.",
   "engines": {
-    "node": "^12 || ^14 || ^16"
+    "node": "^16",
```

@dannobytes You mind putting at least Node 14 back in this? We're still mostly on Node 14 everywhere. When we eventually upgrade the main readme app we can remove 14 then.

```suggestion
    "node": "^14 || ^16"
```

---

For sure. Yeah, I wasn't sure how necessary this was, but considering our other repos still require 14, totally makes sense.

---

Just pushed up!